src/requests/cache/flush_cache.rs

```diff
@@ -0,0 +1,63 @@
+use momento_protos::control_client;
+use tonic::Request;
+
+use crate::requests::cache::MomentoRequest;
+use crate::{utils, CacheClient, MomentoResult};
+
+/// Request to flush a cache of its data
+///
+/// # Arguments
+///
+/// * `name` - The name of the cache to be flushed.
+///
+/// # Examples
+/// Assumes that a CacheClient named `cache_client` has been created and is available.
+/// ```
+/// # fn main() -> anyhow::Result<()> {
+/// # use momento_test_util::create_doctest_cache_client;
+/// # tokio_test::block_on(async {
+/// use momento::requests::cache::flush_cache::FlushCache;
+/// use momento::requests::cache::flush_cache::FlushCacheRequest;
+/// # let (cache_client, cache_name) = create_doctest_cache_client();
+///
+/// match cache_client.flush_cache(cache_name.to_string()).await {
+///     Ok(_) => println!("Flushed cache: {}", cache_name),
+///     Err(e) => eprintln!("Error flushing cache: {}", e),
+/// }
+/// # Ok(())
+/// # })
+/// # }
```

Comment:
We should look into rust doc features if there is hyperlinking or inlining in place so we don't have to repeat the code examples in multiple places (eg here and the cache client)

Reply:
I think there's a way to [link things by name](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html). I imagine we can probably link from the CacheClient to the individual request objects and provide different code examples that are relevant to each (e.g. the common use case call in the cache client and the `send_request` example in the request objects).

Will update here for FlushCache and do a sweep of that when working on https://github.com/momentohq/dev-eco-issue-tracker/issues/724 too

Reply:
Thanks. I look forward to seeing it!

Reply:
yeah that sounds awesome, ty